src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+}
+
+body {
+  margin: 0;
+}
+
+.main__card {
+  display: flex;
+  flex-direction: column;
+  padding: 14px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.main__img {
+  display: flex;
+  justify-content: center;
+  height: 134px;
+  margin: 16px 5px 45px;
+}
+
+.main__card-content {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.main__card-productCode {
+  margin-bottom: 15px;
+  line-height: 13px;
+}
+
+.main__card-reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.main__card-reviews-stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.main__card-star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.main__card-star--active {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.main__card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+
+.main__card-value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.main__card-btn {
```

Comment:
Where do you use this class? I see only `.main__card-btn-link`
![1](https://user-images.githubusercontent.com/66570142/113163871-3edef880-9249-11eb-8ae9-87090644bd41.png)

It looks like you are repeating yourself